README.md

```diff
@@ -5,6 +5,16 @@ A repository for Kustomize manifests
 
 `go get -u github.com/kubernetes-sigs/kustomize`
 
+## List of Kubeflow components available
+
+* Ambassador
+
+* Argo
+
+* Jupyter
+
+* Profiles
+
 ## Basic Usage
```

Comment:
We should probably add a test framework - https://github.com/kubernetes-sigs/kustomize#a-great-report-is-a-bug-reproduction-test